frontend/astro.config.mjs

```diff
@@ -27,7 +31,7 @@ export default defineConfig({
     ],
     excludeRequestHeaders: [
       // For hosting on multiple servers, block the host header
-      'host'
+      //'host'
```

Comment:
remove this line?

Reply:
I think I want to leave it so people know the exact header to block